src/main/java/com/sublinks/sublinksapi/authorization/services/AclService.java

```diff
@@ -246,18 +245,18 @@ private void checkRolePermission() {
 
       if (this.person != null) {
         if (community != null) {
-          // Only permitted if the authorized action allowed in the banned role
-          this.isPermitted = rolePermissionService.isPermitted(person,
-              (RolePermissionInterface) this.authorizedActions, community.getId());
+          this.isPermitted = this.authorizedActions.stream()
+              .allMatch(x -> rolePermissionService.isPermitted(this.person, x, community.getId()));
```

Comment:
Please don't use `x` like this. It makes it harder to follow along with what is happening. Use a more descriptive variable name instead